client/state/preview/reducer.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import assign from 'lodash/assign';
```

Comment:
Because, as I found out relatively recently, `Object.assign` is ES6 and (at least last time I checked) we don't support polyfills for prototype methods in Calypso. So if I use `Object.assign()` then it will only work in browsers that support that feature.